x-pack/plugins/infra/public/components/infrastructure_node_metrics_tables/shared/types.ts

```diff
@@ -21,3 +21,24 @@ export interface SourceProviderProps {
 export type IntegratedNodeMetricsTableProps = UseNodeMetricsTableOptions &
   SourceProviderProps &
   CoreProvidersProps;
+
+export type NodeMetricsTableData<NodeMetricsRow> =
```

Comment:
Because it's more descriptive. In the same way that we strive to choose good names for function arguments we should choose descriptive names for type arguments too. A function with an argument name if `t` would be a code smell (except in specific mathematical domains maybe).

Reply:
Agreed!